plugins/aws-appmesh/config/netconfig.go

```diff
@@ -98,21 +98,26 @@ func New(args *cniSkel.CmdArgs) (*NetConfig, error) {
 }
 
 // validateConfig validates network configuration.
-func validateConfig(config netConfigJSON) error {
+func validateConfig(config NetConfigJSON) error {
 	// Validate if all the required fields are present.
 	if config.IgnoredGID == "" && config.IgnoredUID == "" {
 		return fmt.Errorf("missing required parameter ignoredGID or ignoredUID")
 	}
-	if len(config.AppPorts) == 0 {
-		return fmt.Errorf("missing required parameter appPorts")
-	}
+
 	if config.ProxyEgressPort == "" {
 		return fmt.Errorf("missing required parameter proxyEgressPort")
 	}
-	if config.ProxyIngressPort == "" {
+
+	// AppPorts and ProxyIngressPort go in pairs
```

Comment:
nit: a couple of comments are missing period at the end. We are trying to make sure all comments are well formatted in this repo.